### PR TITLE
Feature/july fifteen notes

### PR DESCRIPTION
**Pull request by TheLarkInn**
Added July 15's notes.

### CODE REVIEW

**Comment by TheLarkInn** [comment]
@webpack/collab-team Feel free to make any ammendments etc.

**Comment by TheLarkInn** [comment]
Planning on leaving this open over the weekend and announce on Monday so it doesn't slip by through the weekend.

**Comment by jhnns** [comment]
Looks good 👍

File: 2016-07/july-15.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#### Core Loaders and Plugins
+
+* Juho has created a document with initial sorting and assignment for loaders and plugins. Sean mentioned that he will go through this list and add anyone else that remains. Once this list is approved we will send to Tobias to extend invitations to the organization and repos.
+
+#### CSS Loader & CSS Modules 
+
+* Johannes brought up that there are still a number of performance related issues in regards to `css-loader` and inquired whether or not it would be possible to isolate the css-modules code from the loader. Tobias said that the changes he had made which were thought to increase performance were not as good as promised. Johannes asked if we could instead have two separate loaders so that people who do not want to use css-modules can simply use a faster lightweight css-loader option instead. Tobias said it may be possible and would look into it.
+
+### Takeaways
+* Finalize vetting people for core loaders and plugins (Sean)
```
Comment (andreicek) [inline 2016-07/july-15.md:48]: Do you have any info on when this will be?

Comment (TheLarkInn) [inline 2016-07/july-15.md:48]: We should have the approval and First list of members agreed on in the next meeting.

**Pull request merged**